modules/configuring-firewall.adoc

```diff
@@ -102,7 +102,7 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 .16+|AWS
 |`aws.amazon.com`
 |443
-|Used to install and manage clusters in an AWS environment.
+|Used to install and manage clusters in an AWS environment. If you use the AWS Security Token Service (STS) with the Cloud Credential Operator (CCO), you must provide access to link:https://docs.openshift.com/container-platform/4.15/authentication/managing_cloud_provider_credentials/cco-short-term-creds.html[CloudFront].
```

Comment:
This is an internal cross-reference for which we need to use `xref` instead of `link`.  And, unfortunately, xrefs cannot be used in modules, only in assemblies. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

```suggestion
|Used to install and manage clusters in an AWS environment. If you use the AWS Security Token Service (STS) with the Cloud Credential Operator (CCO), you must provide access to CloudFront.
```